syntax = "proto2";

message ChangeCurrentSessionDayHumidityMessage {
    required uint32 dayNumber = 2;
    required float humidity = 3;
}
